data/sree-hari-s.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Sreehari S",
+  "displayStatsPublic": true,
```

Comment:
Hi, this line is no longer supported, consider removing it while taking a look a look at the Quick Start guide [here](https://linkfree.io/docs/quickstart-json) if you haven't already.
```suggestion

```